config/aadpodidentity/aad-pod-identity-deployment.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: azureassignedidentities.aadpodidentity.k8s.io
+spec:
+  group: aadpodidentity.k8s.io
+  version: v1
+  names:
+    kind: AzureAssignedIdentity
```

Comment:
`AzureAssignedIdentity` crd isn't required for managed mode. 

The deploy yaml for managed mode: https://github.com/Azure/aad-pod-identity/blob/v1.6.3/deploy/infra/managed-mode-deployment.yaml

Reply:
will remove